Gemfile

```diff
@@ -9,8 +9,8 @@ gem 'rails', '~> 7.0.4', '>= 7.0.4.2'
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
 gem 'sprockets-rails'
 
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
+# Use postgresql as the database for Active Record
+gem 'pg', '~> 1.1'
```

Comment:
Great job @Nowembabazi 🤝

- [ ] Just like the previous reviewer suggested, I keep getting this error whenever I try to run the tests.
![error](https://user-images.githubusercontent.com/63917725/220661476-c2e93f15-f78c-4b35-bbf6-088da64628a8.PNG)